### ISSUE TITLE
Annotation markers are mispaced on the UI

### ISSUE THREAD

**Issue by jonatan-ivanov** [openzipkin/zipkin#3356]
When I annotate spans, the "dot" that shows them on the timeline (see the screenshot) does not seem to be where it should.

<img width="960" alt="annotation screen shot" src="https://user-images.githubusercontent.com/3044070/124312737-a6e0fb80-db24-11eb-9570-27fbb634d90b.png">

As you can see the "dot" is in the very beginning while it's relative time is 4s and if you check the duration, it should around the middle section of the bar.

This is my trace, as you can see it started at `1625248613237960` and the annotation happened at `1625248616245438`: `(1625248616245438 - 1625248613237960) / 1000 / 1000` = 3s.

```json
[
    {
        "traceId": "1c4ac61d32fce8f0",
        "parentId": "1c4ac61d32fce8f0",
        "id": "39345d9b9b671972",
        "name": "repair",
        "timestamp": 1625248613237960,
        "duration": 10031051,
        "localEndpoint": {
            "serviceName": "example-service",
            "ipv4": "192.168.0.112"
        },
        "annotations": [
            {
                "timestamp": 1625248616245438,
                "value": "new-tool"
            }
        ],
        "tags": {
            "author": "Joshua Bloch",
            "isbn": "978-0134685991",
            "title": "Effective Java"
        }
    },
    {
        "traceId": "1c4ac61d32fce8f0",
        "id": "1c4ac61d32fce8f0",
        "name": "buy",
        "timestamp": 1625248612208230,
        "duration": 11082344,
        "localEndpoint": {
            "serviceName": "example-service",
            "ipv4": "192.168.0.112"
        },
        "tags": {
            "author": "Joshua Bloch",
            "isbn": "978-0134685991",
            "title": "Effective Java"
        }
    }
]
```

I'm running zipkin in docker, this way: [`docker-compose.yml`](https://github.com/jonatan-ivanov/local-services/blob/main/zipkin/docker-compose.yml), I'm using the latest image from DockerHub:
```
"org.opencontainers.image.created": "2020-12-25",
"org.opencontainers.image.revision": "7bf3aab83"
```

**Comment by jcchavezs** [comment]
Could you please check this @tacigar? Cc @basvanbeek

**Comment by tacigar** [comment]
This was fixed by #3486 🙇 

![Screenshot 2022-12-26 9 52 41](https://user-images.githubusercontent.com/19551419/209486755-110d46f5-1e16-4fe2-bd3f-2ea1faf4e744.png)
